app/src/main/java/ch/epfl/skysync/models/user/Admin.kt

```diff
@@ -16,4 +16,8 @@ data class Admin(
   override fun addRoleType(roleType: RoleType): Admin {
     return this.copy(roleTypes = roleTypes + roleType)
   }
+
+    override fun toString(): String {
```

Comment:
In my opinion overriding the `toString` method to only return the name is misleading, instead you could call it `name`,
also why define it in all class that implements `User` but not in `User` directly ?